app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/labtest/LabTestScreen.kt

```diff
@@ -0,0 +1,394 @@
+
+    val title = labTestDetail.bannerHeader?.let { stringResource(it) } ?: return
+    val body = labTestDetail.bannerText?.let { stringResource(it) } ?: return
+    val clickableText = labTestDetail.bannerClickableText?.let { stringResource(it) }
+
+    Row(
+        Modifier
+            .fillMaxWidth()
+            .padding(vertical = 16.dp)
+            .background(color = bannerBackgroundBlue)
+            .padding(16.dp)
+    ) {
+        DecorativeImage(resourceId = R.drawable.ic_info)
+        Spacer(modifier = Modifier.size(8.dp))
+        Column {
+            Text(
+                text = title,
+                color = blue,
+                style = MyHealthTypography.body2.bold()
+            )
+
+            clickableText?.let {
+                MyHealthClickableText(
+                    style = MyHealthTypography.body2.copy(color = blue),
+                    fullText = body,
+                    clickableText = it,
+                    action = onClickFaq,
+                    clickableStyle = SpanStyle(
+                        color = primaryBlue,
+                        textDecoration = TextDecoration.Underline,
+                        fontWeight = FontWeight.Bold
+                    )
+                )
+            } ?: run {
+                Text(
+                    style = MyHealthTypography.body2.copy(color = blue),
+                    text = body,
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun LabTestUi(labTestDetail: LabTestDetail, onClickFaq: () -> Unit) {
+    LabTestHeaderUi(
+        labTestDetail.header,
+        labTestDetail.summary?.let { stringResource(id = it) },
+        onClickFaq
+    )
+
+    LabTestItemUi(
+        labTestDetail.title1,
+        labTestDetail.testName.orPlaceholderIfNullOrBlank(),
+    )
+
+    LabTestResultItemUi(labTestDetail)
+
+    LabTestItemUi(
+        labTestDetail.title3,
+        labTestDetail.testStatus?.let { stringResource(id = it) },
+    )
+
+    LabTestDivider()
+}
+
+@Composable
+private fun LabOrderUi(labTestDetail: LabTestDetail) {
+    LabTestItemUi(
+        labTestDetail.title1,
+        labTestDetail.timelineDateTime.orPlaceholderIfNullOrBlank(),
+    )
+
+    LabTestItemUi(
+        labTestDetail.title2,
+        labTestDetail.orderingProvider.orPlaceholderIfNullOrBlank(),
+    )
+
+    LabTestItemUi(
+        labTestDetail.title3,
+        labTestDetail.reportingSource.orPlaceholderIfNullOrBlank(),
+    )
+
+    LabTestDivider()
+}
+
+@Composable
+private fun LabTestHeaderUi(title: Int?, body: String?, onClickFaq: () -> Unit) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(top = 16.dp)
+    ) {
+        title?.let {
+            Text(
+                text = stringResource(id = it),
+                style = MyHealthTypography.body2.bold()
+            )
+        }
+
+        body?.let {
+            MyHealthClickableText(
```

Comment:
I would recommend usage of HGTextButton

Reply:
This is not a button, it's a long Text that contains a clickable link, like Spannable on TextViews